lib/granite/action/transaction_manager.rb

```diff
@@ -67,7 +73,7 @@ def trigger_callbacks
 
           callback_listeners.reverse.each do |listener|
             begin
-              listener.run_callbacks :commit
+              listener.respond_to?(:run_callbacks) ? listener.run_callbacks(:commit) : listener.call
```

Comment:
Is a symbol accepted as a `listener`?

Reply:
no. object or block

Reply:
According to [the docs](https://github.com/toptal/granite/blob/2d4149bff76e502b6ce6db05b69a4572c4504e43/lib/granite/action/transaction.rb#L18):
> Uses the same arguments as `ActiveSupport::Callbacks.set_callback` except for the first two

While [`set_callback` docs](https://api.rubyonrails.org/v5.1/classes/ActiveSupport/Callbacks/ClassMethods.html#method-i-set_callback) state that:
> The callback can be specified as a symbol naming an instance method

I believe the support for symbols should be added or else the docs should be fixed to reflect the fact that we don't support symbols.

Reply:
You read docs for a different class. symbols work fine for BAs. TransactionManager doesn't need symbols support and doesn't claim to support them.